Modules/System/Translation/src/TranslationImplementation.Codeunit.al

```diff
@@ -87,6 +89,37 @@ codeunit 3712 "Translation Implementation"
         DeleteTranslations(RecVariant, Translation);
     end;
 
+    procedure Copy(FromRecVariant: Variant; ToRecVariant: Variant)
+    begin        
+        CopyTranslations(FromRecVariant, ToRecVariant, 0);
+    end;
+    procedure Copy(FromRecVariant: Variant; ToRecVariant: Variant; FieldId: Integer)
+    begin
+        if FieldId = 0 then
+            Error(FieldIdZeroErr);
+        CopyTranslations(FromRecVariant, ToRecVariant, FieldId);
+    end;
+
+    local procedure CopyTranslations(FromRecVariant: Variant; ToRecVariant: Variant; FieldId: Integer)
```

Comment:
you can rename this method to Copy (and remove the other two Copy functions) and call this directly from the facade.
As 0 is an accepted value, there is no need to check and throw an error. The error would not be actionable by a user, it would always be a developer error.